docs/reference/lidar.md

```diff
@@ -292,6 +292,8 @@ period = wb_lidar_get_sampling_period(tag)
 *enable and disable lidar updates*
 
 The `wb_lidar_enable` function allows the user to enable lidar updates.
+Once the lidar is enabled, it will copy range images from VRAM to RAM at each time step, regardless of `wb_lidar_get_range_image` calls.
```

Comment:
```suggestion
Once the lidar is enabled, it will copy range images from GPU memory to CPU memory at each time step, regardless of `wb_lidar_get_range_image` calls.
```